doc/howToBuildADuck.md

```diff
@@ -51,6 +55,8 @@ PapaDuck is the final Duck in the ClusterDuck and transmits network data to the
 
 You will need the [Arduino IDE](https://www.arduino.cc/en/main/software) to install the firmware on your Arduino devices.
 
+Or you can use your favourite Code editor with the [platformIO](https://platformio.org/) plugin.
```

Comment:
American English is "favorite" without the "u"

Reply:
solved